codelabs/keyboard_navigation/keyboard_navigation.md

```diff
@@ -351,11 +354,12 @@ Open the playground and enter into keyboard navigation mode (**ctrl + shift + k*
 on to the workspace and navigate to the first block. From here hit the **S** key
 to go to the next block. Notice how the cursor skips over the previous and next
 connection and goes straight to the next block.
+
 ![](./new_cursor.gif)
 
 ## Add a shortcut
 In this section we add a shortcut to allow users to move their
-cursor to the top of the stack their cursor is currently on.
+cursor to the top of the stack their cursor is currently on when they hit **shift + T**.
```

Comment:
Wouldn't shift (UP_ARROW) be better? or shift (W)?. 
T is reserved or toolbox, shift T to go to top might be confusing.

Reply:
I could do ctrl + W. Shift + w is used for moving around the workspace